ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -134,13 +134,16 @@ monitoring_pod="$( oc --insecure-skip-tls-verify get pods -n openshift-monitorin
 queue ${ARTIFACT_DIR}/metrics/prometheus.tar.gz oc --insecure-skip-tls-verify exec -n openshift-monitoring "${monitoring_pod}" -- tar cvzf - -C /prometheus .
 
 cat >> ${SHARED_DIR}/custom-links.txt << EOF
+<p>
```

Comment:
```suggestion
<p>
<span id="promecieus_link"></span>
```
and later on
```
document.getElementById("wrapper").append(a);
```
replaced with
```
document.getElementById("promecieus_link").append(a);
```
so that the link would be added below `<p>`